schedule/yast/dud_development_tools.yaml

```diff
@@ -40,6 +39,6 @@ schedule:
 test_data:
   dud_repos:
     # The URI is the same as set in 'data/dev_tools.dud' file.
-    - URI: 'ftp://openqa.suse.de/SLE-15-Module-Development-Tools-POOL-x86_64-Media1-CURRENT'
+    - URI: 'ftp://openqa.suse.de/SLE-15-SP3-Module-Development-Tools-POOL-x86_64-Build36.5-Media1'
```

Comment:
it is not clear in my head at the moment, but this could also get some benefits using the `addons` variable. So we do not need to have it static. but it needs some research.

Reply:
Good point, as we have variable which contains whole thing after `p://openqa.suse.de/`, and there is variable `'REPO_HOST` too. Also, keep in mind that if we use `vars` in yaml, those variables also get expanded.